### PR TITLE
Update code review README language to include remote workers

### PR DESCRIPTION
**Pull request by robwierzbowski**
I've seen this document referenced often by other developers. Would you consider merging this change to put remotes on equal footing with in-house people during code reviews?

### CODE REVIEW

**Comment by robwierzbowski** [comment]
Used the GH editing interface to make changes. Will happily squash with any additional requests.

**Comment by derekprior** [comment]
I like this.

**Comment by gerrywastaken** [comment]
I always took it to mean as you have written and naturally did the translation in my head. But I agree, the current wording could be misinterpreted.

**Comment by robwierzbowski** [comment]
Great. In companies that are just getting started with remote (like the one I'm working with now) I think it's good to be explicit. Any additional suggestions?

File: code-review/README.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ Everyone
 * Don't use sarcasm.
 * Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
   If they are, use them with aplomb.
-* Talk in person if there are too many "I didn't understand" or "Alternative
-  solution:" comments. Post a follow-up comment summarizing offline discussion.
+* Talk in chat, screenshare, or in person if there are too many "I didn't
```
Comment (jferris) [inline code-review/README.md]: I suggest re-ordering these to put "in-person" first. If it's available, it's both low-cost and high-bandwidth.

Comment (robwierzbowski) [inline code-review/README.md]: > * Talk in person, in chat, or in a screenshare if there are too many "I didn't understand" or "Alternative solution:" comments. Post a follow-up comment summarizing the discussion.

Thoughts?

Comment (robwierzbowski) [inline code-review/README.md]: As a sometimes remote worker / remote advocate, I like to put the remote options first to level the perceived value. I think no matter the order, people will self select what makes the most sense for them. Even when I'm in office I often do a quick pair with Screenhero to walk people through code.

**Comment by robwierzbowski** [comment]
Happy to make any changes to get this in sometime soon. No hurry, I'm not in your repos very often, just don't want to forget 🐘 .

File: code-review/README.md (hunk 1 above)
Comment (mike-burns) [inline code-review/README.md]: The common thing we're trying to express here is: communicate synchronously. Should we state that instead?

> Talk synchronously (e.g. chat, screensharing, in person) if there are too many "I didn't understand" or "Alternative solution:" comments. Post a follow-up comment summarizing the discussion.

Comment (robwierzbowski) [inline code-review/README.md]: Luv it, will update.

**Comment by robwierzbowski** [comment]
Back to you. Feel free to squash on merge into that first commit.

**Comment by mike-burns** [comment]
Committed as 083c9edeb87159866490c951070ef6fc4b8a04fb, thank you.

**Comment by robwierzbowski** [comment]
Love it! Thanks.

On Friday, May 27, 2016, Mike Burns <notifications@github.com> wrote:

> Closed #423 <https://github.com/thoughtbot/guides/pull/423>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/thoughtbot/guides/pull/423#event-674208174>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AA4gJzh0fl7PL0Qn9pojxM3p-f7SG8rPks5qFrLtgaJpZM4IaKA2>
> .
>


-- 
Rob Wierzbowski
@robwierzbowski <http://twitter.com/#!/robwierzbowski>
http://github.com/robwierzbowski
http://robwierzbowski.com

**Pull request closed without merging**